app/controllers/v1/mixtapes_controller.rb

```diff
@@ -0,0 +1,14 @@
+class V1::MixtapesController < V1::ApplicationController
+
+  def index
+
+  end
+
+  def create()
```

Comment:
Well this user is (hypothetically) passing some parameters back with which the resource is being generated.

Which makes me think that this is a POST request, rather then a GET request.